packages/dev/core/src/Loading/Plugins/babylonFileLoader.ts

```diff
@@ -803,7 +808,7 @@ SceneLoader.RegisterPlugin({
                 for (let index = 0, cache = scene.meshes.length; index < cache; index++) {
                     currentMesh = scene.meshes[index];
                     if (currentMesh._waitingParentId) {
-                        const parent = scene.getLastEntryById(currentMesh._waitingParentId);

```

Comment:
can we get rid of _waintingParentId then?

Reply:
I don't think we can, because the nodes still need a way to store the id that was on the JSON while we're still loading all the nodes